Day-I-Social-Card/social-card/src/components/HeaderComponents/HeaderTitle.js

```diff
@@ -1,2 +1,19 @@
 import React from 'react';
-import './Header.css';
+import HeaderContent from "./HeaderContent";
+import "./Header.css";
+import moment from 'moment';
```

Comment:
you don't have moment installed in package.json, so this import breaks the code.